event_registration_partner_unique/tests/test_event.py

```diff
@@ -1,22 +1,24 @@
 # Copyright 2016 Antiun Ingeniería S.L. - Jairo Llopis
 # Copyright 2020 Tecnativa - Víctor Martínez
+# Copyright 2023 Tecnativa - Carolina Fernandez
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
 from odoo.exceptions import ValidationError
 from odoo.tests.common import TransactionCase
 
 
 class DuplicatedPartnerCase(TransactionCase):
-    def setUp(self):
-        super().setUp()
-        self.event = self.env.ref("event.event_0")
-        self.event.forbid_duplicates = False
-        self.partner = self.env.ref("base.res_partner_1")
-        self.registration = self.env["event.registration"].create(
+    @classmethod
+    def setUpClass(cls):
+        super(DuplicatedPartnerCase, cls).setUpClass()
```

Comment:
No need to explicitly  pass the class and you instance if they're the same
```suggestion
        super().setUpClass()
```